src/layouts/default.vue

```diff
@@ -106,6 +98,28 @@ export default {
       description: state => state.description,
       action:      state => state.action
     }),
+    defaultRoutes() {
+      return [
+        '/General',
+        '/K8s',
+        '/PortForwarding',
+        '/Images',
+        '/Troubleshooting'
+      ];
+    },
+    /**
+     * Inserts host preferences route at array index 1
+     */
+    modifiedRoutes() {
+      return [
+        ...this.defaultRoutes.slice(0, 1),
+        '/HostPreferences',
+        ...this.defaultRoutes.slice(1)
+      ];
+    },
+    routes() {
+      return os.platform().startsWith('win') ? this.defaultRoutes : this.modifiedRoutes;
```

Comment:
We're back to modifying routes based on platform.. I anticipate this will go away again after we add more settings to `App/Host` preferences